### DISCUSSION TITLE
Interactions only works when clicking on another story

### DISCUSSION THREAD

**Discussion by gracekiim** [storybookjs/storybook#17667] [Help]
On first load, `canvasElement` seems to be an empty div, and only when clicking on another story do they work:

https://user-images.githubusercontent.com/59268038/157359769-d7dc2a77-4f37-4b6d-940e-38f672393976.mp4

I saw these issues:

- https://github.com/storybookjs/storybook/issues/16430
- https://github.com/storybookjs/storybook/issues/16872
- https://github.com/storybookjs/storybook/issues/16492

And was wondering if it could be related to them?

**Comment by shilman** [comment]
Do you a have a reproduction repo you can share? If not, can you create one? See [how to create a repro](https://storybook.js.org/docs/react/contribute/how-to-reproduce). 🙏

**Reply by robertandradelifeio** [reply]
Hi! We're having the same issue, here is our repro repo https://github.com/alysiadillon/chromatic-demo

**Reply by robertandradelifeio** [reply]
Here is the link to the hosted version https://629027bc62c266004a7e0621-snokjylvic.chromatic.com/?path=/story/example-page--logged-in
